pkcs11/C/pkcs11_sample_attributes.c

```diff
@@ -501,7 +510,22 @@ int main(int argc, char *argv[])
         loggedIn = 1;
         printf("Successfully logged in. \n");
 
-        if (symmetric == 0)
+		if (ksid_type = keyIdAttr)

```

Comment:
This if condition is incorrect as it will be always true due to use of assignment operator.